### DISCUSSION TITLE
Clock source setting: is "16 MHz (PLL)" the same as internal?

### DISCUSSION THREAD

**Discussion by RossAWaddell** [SpenceKonde/ATTinyCore#878] [Support / Q & A]
I'm using an ATtiny85 to run some NeoPixels using the Adafruit_NeoPixel.h library which apparently will only work with a clock source of 16 MHz. After installing ATTinyCore I see a lot of choices for the Clock Source but can't work out if "16 MHz (PLL)" means the internal one?

**Comment by hmeijdam** [comment]
Yes the Attiny85 has an internal 16MHz clock option.

**Reply by RossAWaddell** [reply]
So that is what I should select to set the fuses via Burn Bootloader to get 16 MHz internal clock option?
